adapters/rubicon/rubicon.go

```diff
@@ -750,10 +736,8 @@ func updateImpRpTargetWithFpdAttributes(extImp rubiconExtImpBidder, extImpRubico
 	return updatedTarget, nil
 }
 
-func extractDfpAdUnitCode(data rubiconData, contextData rubiconData) string {
-	if contextData.AdServer.Name == "gam" && contextData.AdServer.AdSlot != "" {
-		return contextData.AdServer.AdSlot
-	} else if data.AdServer.Name == "gam" && data.AdServer.AdSlot != "" {
+func extractDfpAdUnitCode(data rubiconData) string {
+	if data.AdServer.Name == "gam" && data.AdServer.AdSlot != "" {
```

Comment:
How about `if strings.ToLower(data.AdServer.Name) == "gam"` to make sure "GAM" is also valid Ad Server Name? 
Optional: `&& len(data.AdServer.AdSlot) != 0`, it just looks cleaner to me.

Also please sync this branch with the latest PBS main branch.

Reply:
"gam": this is exactly how it works in PrebidJava, not sure if case matters but i would prefer to copy java logic